frontend/src/services/fetchData.ts

```diff
@@ -125,3 +126,20 @@ export const fetchDelegatedAdmins = async (
     return delegatedAdmins;
 
 };
+
+/**
+ * This will call api to delete delegatedAdminPrivilege record from backend and fetch again
+ * to refresh the state.
+ * @param accessPrivilegegId id to delete delegated admin accesss record.
+ */
+export const deleteAndRefreshDelegatedAdmin = async (
+    accessPrivilegegId: number
+// ): Promise<FamAccessControlPrivilegeGetResponse[]> => {
```

Comment:
just wondering why we comment out this line? thanks!!

Reply:
It should have been removed. Thanks!

Reply:
I think we should keep it, other delete and refresh method all define the return type

Reply:
I added the return type